### ISSUE TITLE
Time

### ISSUE THREAD

**Issue by FauconFan** [IP7/concours_runner#4]
A un moment, j'avais dit que récupérer le temps d'éxécution du programme pouvait être compliqué. Il se trouve que ça peut se faire si on construit des images docker particulières. Il faut que j'écrive et que je test pas mal de trucs...

Je veux bien le faire, mais je vais pas le faire, si on compte pas faire des tests avec une limite de temps...

Donc ma question, est-ce que je le fais ? (tout simplement x))

**Comment by nobrakal** [comment]
Ça n'est sans doute pas une priorité, la seule chose à avoir serait un timeout, l'histoire de ne pas autoriser les solutions qui sont clairement des trucs pas viables (genre une boucle infinie)

**Comment by Peaupote** [comment]
Je suis d'accord. D'autant que des limites de temps absolues n'ont pas vraiment de sens si on autorise un peu tous les langages. 

On s'en occupera quand le reste sera fait.

**Comment by FauconFan** [comment]
Ok ça marche, je sais pas comment faire le timeout, autrement qu'en fixant une règle, du type, si ça dure plus de 3 secondes on arrête tout. Ca vous va ?
